ppr-ui/src/components/mhrRegistration/HomeLocation/HomeLocationType.vue

```diff
@@ -244,16 +246,16 @@ export default defineComponent({
       isValidHomePark: false,
       isVerifyingPid: false,
       isValidLocationInfo: false,
-      locationTypeOption: HomeLocationTypes,
-      otherTypeOption: HomeLocationTypes,
-      dealerManufacturerLot: '',
-      homeParkName: '',
-      homeParkPad: '',
-      pidNumber: '',
+      locationTypeOption: getMhrRegistrationLocation.value?.locationType || HomeLocationTypes,
```

Comment:
There must be a way to destructure it without calling getter for each.

Reply:
Yea, i could look more into this. 
Right now the local property names are slightly different than the api property names, might require some refactoring to get those in sync, also to map only the select values.
Will 👀

Reply:
Yea not a biggie, im ok either way. I was thinking that props dont have to map directly, just destructure and reuse `const { locationType, otherType, .... } = getMhrRegistrationLocation.value`

Reply:
Oh sorry, i misunderstood!
Gotcha, updated :)